SPECS/procps-ng/procps-ng.spec

```diff
@@ -55,10 +57,21 @@ ln -s %{_bindir}/pidof %{buildroot}%{_sbindir}/pidof
 find %{buildroot} -type f -name "*.la" -delete -print
 %find_lang %{name} --all-name --with-man
 
+install -Dm 644 %{SOURCE1} %{buildroot}%{_sysconfdir}/sysctl.d/10-console-messages.conf
```

Comment:
@ddstreetmicrosoft would this be better supplied by `systemd` since `systemd` owns `/etc/sysctl.d`? I could also see this eventually live in `azurelinux-profile` packages, but these are not ready/available yet.

Reply:
as `procps-ng` doesn't provide any configuration at all currently, it probably doesn't make a lot of sense to put this here. Putting it into `systemd` or `azurelinux-release` (or `azurelinux-release-$VARIANT` if we want to add that) probably makes sense. It would be good to have a better overall design of where we want config like this, but for now probably `systemd` is the best place to put it, as you suggested.

Note that it should go into `%{_prefix}/lib/sysctl.d/`, and not into `%{_sysconfdir}/sysctl.d`. Distro-provided config should go into `/usr/lib` while `/etc` is normally reserved for end-user-provided config.